Buildings/Experimental/DHC/CentralPlants/Cooling/Examples/Plant.mo

```diff
@@ -108,9 +108,11 @@ model Plant
     annotation (Placement(transformation(extent={{60,-20},{40,0}})));
 equation
   connect(dpMea.y,pla.dpMea)
-    annotation (Line(points={{-39,-50},{-20,-50},{-20,13.2667},{-10.6667,13.2667}},color={0,0,127}));
+    annotation (Line(points={{-39,-50},{-20,-50},{-20,13.2667},{-10.6667,
+          13.2667}},                                                               color={0,0,127}));
```

Comment:
Please fix this format issue.

Reply:
The format is right in Dymola text, but I revised it through Notepad++.